stubs/chevron/METADATA.toml

```diff
@@ -0,0 +1,2 @@
+version = "0.14.*"
+python2 = true
```

Comment:
Do you have a particular reason to support Python 2? It's fine, but we don't require the stubs to support Python 2 even if the library itself supports Python 2.

Reply:
Not really, I thought that this was to indicate that the library itself supports python 2. Removing it 👍.